source/chapters/advanced_topics.rst

```diff
@@ -1402,11 +1491,15 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
 
    Doubles :mixxx:coref:`beatloop_size <[ChannelN],beatloop_size>`. If :mixxx:coref:`beatloop_size <[ChannelN],beatloop_size>` equals the size of the loop, the loop is resized.
 
+   If a saved loop is currently enabled, the modification is saved to the hotcue slot immediately.
```

Comment:
Be aware, that this will destroy a saved loop if the you exit it using the loop having effect.

Reply:
I don't understand you you mean. Can you rephrase?

Reply:
I was just aware of the following:

Sometimes I enable a loop in the outro, bring in the new track in and the think the loop by having it, before if fade out the last track.

This "destroyes" my original saved loop.
This is nothing new with normal loops, but for this use case saved loops are currently useless.

I wonder how competitors do it.

Reply:
Serato has a concept of "locked" loops that don't have this behavior.